libraries.json

```diff
@@ -13,53 +19,212 @@
 		"description": "Meteor.js is a full stack framework. It means that you can control everything from the database to the UI with the same language and the same framework, and it is one of Meteor’s primary advantages.",
 		"image": "https://user-images.githubusercontent.com/841294/26841702-0902bbee-4af3-11e7-9805-0618da66a246.png",
 		"language": "JavaScript"
 	},
+	"MobX": { 
+		"imports": ["mobx"],
+		"technologies": ["Web Development", "SPA"],
+		"description": "MobX is a battle tested library that makes state management simple and scalable by transparently applying functional reactive programming (TFRP)",
+		"language": "JavaScript"
+	},
+	"NativeScript": {
+		"imports": ["nativescript"],
+		"technologies": ["Mobile applications"],
+		"description": "Open source framework for building truly native mobile apps with Angular, Vue.js, TypeScript, or JavaScript.",
+		"language": "JavaScript"
+	},
+	"NextJS": {
+		"imports": ["nextjs"],
+		"technologies": ["Web Development", "SPA", "Server side rendering"],
+		"description": "Production grade React applications that scale. The world's leading companies use Next.js to build server-rendered applications, static websites, and more.",
+		"language": "JavaScript"
+	},
+	"NgRx": { 
+		"imports": ["@ngrx/core"],
+		"technologies": ["Web Development", "SPA"],
+		"description": "RxJS powered state management for Angular applications, inspired by Redux",
+		"language": "JavaScript"
+	},
+	"NuxtJS": {
+		"imports": ["nuxtjs"],
+		"technologies": ["Web Development", "SPA", "Server side rendering"],
+		"description": "Nuxt.js is a free and open source web application framework based on Vue.js, Node.js, Webpack and Babel.js.",
+		"language": "JavaScript"
+	},
+	"Parcel": {
+		"imports": ["parcel"],
+		"technologies": ["Build tools"],
+		"description": "Blazing fast, zero configuration web application bundler.",
+		"language": "JavaScript"
+	},
+	"Proton Native": {
+		"imports": ["proton-native"],
+		"technologies": ["Desktop applications"],
+		"description": "Create native desktop applications through a React syntax, on all platforms",
+		"language": "JavaScript"
+	},
 	"ReactJS": {
 		"imports": ["react"],
-		"technologies": ["Web Development", "SPA"],
+		"technologies": ["Web Development", "SPA", "JavaScript Frameworks"],
 		"description": "A JavaScript library for building user interfaces",
 		"image": "https://cdn4.iconfinder.com/data/icons/logos-3/600/React.js_logo-512.png",
 		"language": "JavaScript"
 	},
-	"Svelte": {
+	"React Native": {
+		"imports": ["react-native"],
+		"technologies": ["Mobile applications"],
+		"description": "React Native combines the best parts of native development with React, a best-in-class JavaScript library for building user interfaces.",
+		"language": "JavaScript"
+	},
+	"Redux": { 
+		"imports": ["redux"],
+		"technologies": ["Web Development"],
+		"description": "A predictable state container for JavaScript apps.",
+		"image": "????",
```

Comment:
It would be better to leave out this field if we don't have a URL yet (same with rxjs)